pkgs/development/libraries/boost/generic.nix

```diff
@@ -114,6 +116,13 @@ stdenv.mkDerivation {
   patchFlags = [];
 
   patches = patches
+  ++ optional (versionOlder version "1.70") (fetchpatch {
```

Comment:
I am not sure if there is any order here but maybe put it betweenline 137 and 138?

Reply:
Excellent point; I've moved it.